documentation/mappping-new-iris/README.md

```diff
@@ -0,0 +1,21 @@
+# mapping NEw IRIs
+
+This directory contains the mapping file for renaming the version 1.7 and older CCO ontology elements to the schema intorduced in version 2.0
+It is desiging to work with the ROBOT command `rename`
+
+robot rename -i [file-name-old-iris] --mappings iri-mapping-v2.0.csv --add-prefix "cco2: https://www.commoncoreontologies.org/" --allow-missing-entities true -o [file-name-new-iris]
+
+Ontology IRIs require a manual find and replace.
+
```

Comment:
Should All Core Ontology be added to this list? It has no content, but it does have an IRI that changed.